### PR TITLE
Configuration de tâches Grunt

### PR DESCRIPTION
**Pull request by Neb19**
On y trouve l'ajout des dépendances dans package.json pour l'execution des tâches de Grunt et la configuration minimum pour pouvoir utiliser sass, compiler en css, concaténer et minifier les fichiers css dans gruntfile.css 
&& création du fichier index.html dans www/

### CODE REVIEW

File: .bowerrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "directory" : "www"
+}
```
Comment (pimolo) [inline .bowerrc]: A general good practise is always keeping an empty line at the end of all source files.
That's what this red pictogram means ;)

File: gruntfile.js [hunk 2]
```diff
@@ -3,30 +3,53 @@ module.exports = function(grunt) {
...
+          dest: 'www/css',
+          ext: '.css'
+        }]
       }
     },
+
+    concat: {
+      //options: {
+      //  separator: ';',
+      //},
```
Comment (pimolo) [inline gruntfile.js]: oops

File: www/sass/_mixin.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+/*************************/
```
Comment (pimolo) [inline www/sass/_mixin.scss]: IMO this file should be named `_mixins.scss`. Same with `_vars.scss`. Hey everyone what do you think ? @UnderDevCrew/true-developers  ?

**Comment by pimolo** [comment]
Are sourcemaps enabled? This should be tested. Just write a few lines in a SCSS file, build the CSS with Grunt and check with Chrome DevTools if you can see SCSS properties in the inspector instead of css properties (from the compiled file)

File: .bowerrc (hunk 1 above)
Comment (Neb19) [inline .bowerrc]: thanks pim I didn't know it

**Comment by Neb19** [comment]
Yes sourcemaps are enabled. I checked it before pull request

File: www/sass/_mixin.scss (hunk 3 above)
Comment (girard-g) [inline www/sass/_mixin.scss]: I agree we should have the same standards ..

File: gruntfile.js [hunk 4]
```diff
@@ -3,30 +3,53 @@ module.exports = function(grunt) {
...
-      options: {
-        sourceMap: true
-        },
       dist: {
-        files: {
-          'assets/css/app.css' : 'assets/sass/app.scss'
-        }
+        files: [{
+          expand: true,
+          cwd: 'www/sass',
```
Comment (girard-g) [inline gruntfile.js]: @Neb19  you should put all .css .scss and .js files in a directory like assets/css or assets/js ... instead putting them directly in www/

Comment (pimolo) [inline gruntfile.js]: :+1: good catch

**Comment by pimolo** [comment]
Nope.

[Rejected with PullApprove](https://pullapprove.com/UnderDevCrew/HapplyHourSite/pull-request/1/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=UnderDevCrew/HapplyHourSite)

**Comment by girard-g** [comment]
Nope.

[Rejected with PullApprove](https://pullapprove.com/UnderDevCrew/HapplyHourSite/pull-request/1/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=UnderDevCrew/HapplyHourSite)

**Comment by duncanroze** [comment]
Nope.

[Rejected with PullApprove](https://pullapprove.com/UnderDevCrew/HapplyHourSite/pull-request/1/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=UnderDevCrew/HapplyHourSite)

**Comment by girard-g** [comment]
@Neb19 You should review your code with all advises in comments and PR again

**Comment by pimolo** [comment]
mdr

File: www/index.html [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<!doctype html>
+<html lang="fr">
+<head>
+    <meta charset="UTF-8">
+    <title>Document</title>
+</head>
+<body>
+
+</body>
+</html>
```
Comment (pimolo) [inline www/index.html]: :3

File: .bowerrc (hunk 1 above)
Comment (pimolo) [inline .bowerrc]: :3

File: .gitignore [hunk 6]
```diff
@@ -106,3 +106,5 @@ com_crashlytics_export_strings.xml
 crashlytics.properties
 crashlytics-build.properties
 
+##Sass
+.sass-cache
```
Comment (pimolo) [inline .gitignore]: :3

File: gruntfile.js [hunk 7]
```diff
@@ -3,30 +3,53 @@ module.exports = function(grunt) {
   // Project configuration.
   grunt.initConfig({
     pkg: grunt.file.readJSON('package.json'),
+
     sass: {
-      options: {
+      option: {
```
Comment (pimolo) [inline gruntfile.js]: missing `s`
and according to (the doc)[https://github.com/gruntjs/grunt-contrib-sass#example-config] `options` should be in `dist`

**Comment by pimolo** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/UnderDevCrew/HapplyHourSite/pull-request/1/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=UnderDevCrew/HapplyHourSite)

**Comment by duncanroze** [comment]
:+1: 

[Approved with PullApprove](https://pullapprove.com/UnderDevCrew/HapplyHourSite/pull-request/1/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=UnderDevCrew/HapplyHourSite)

**Pull request merged**